readme.md

```diff
@@ -32,6 +32,7 @@ automatically include outputs of task dependencies in the Docker build context.
 - `name` the name to use for this container, may include a tag
 - `tags` (optional) an argument list of tags to create; any tag in `name` will
   be stripped before applying a specific tag; defaults to the empty set
+  Tag can contains String with GString syntax. In this case value was evaluate after all plugins was load.
```

Comment:
This is a little confusing. Maybe try something like,

> The tag may contain a String with GString syntax. The GString value will evaluate after all plugins are loaded.

Reply:
Indeed, it's less confused. it's good for me